contract/src/agoric-basics.contract.js

```diff
@@ -97,12 +75,21 @@ export const bagPrice = (bag, inventory) => {
 };
 // #endregion
 
-/** @typedef {{[key: string]: {tradePrice: Amount, maxTickets: NatValue}}} Inventory */
+/**
+ * Inventory contains price and maximum for each type of tickets
+ * For example:
```

Comment:
maybe use the [@example](https://jsdoc.app/tags-example) tag? not sure how valuable it is in this context...

```suggestion
 * @example
```